### PR TITLE
Add `.editorconfig` file

### PR DESCRIPTION
**Pull request by snahornyi**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,331 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
```
Comment (rpanchyk) [inline .editorconfig:9]: Will it affect JSON files?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+ij_java_use_single_class_imports = true
+ij_java_variable_annotation_wrap = off
+ij_java_visibility = public
+ij_java_while_brace_force = never
+ij_java_while_on_new_line = false
+ij_java_wrap_comments = false
+ij_java_wrap_first_method_in_call_chain = false
+ij_java_wrap_long_lines = false
+
+[.editorconfig]
```
Comment (rpanchyk) [inline .editorconfig:270]: `editorconfig` for `editorconfig`? oO

File: .editorconfig (hunk 1 above)
Comment (snahornyi) [reply .editorconfig:9]: No

Comment (snahornyi) [reply .editorconfig:9]: We have specific json properties, which gives json lines possibility to be as long as needed

File: .editorconfig (hunk 2 above)
Comment (snahornyi) [reply .editorconfig:270]: Yes, seems this is useful config

**Review by rpanchyk — APPROVED** [review]

**Pull request merged**